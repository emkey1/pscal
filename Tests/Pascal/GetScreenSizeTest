#!/usr/bin/env pascal
program GetScreenSizeTest;

const
  ScreenWidth  = 320;
  ScreenHeight = 240;
  Title        = 'GetScreenSizeTest';

var
  width, height: Integer;
begin
  InitGraph(ScreenWidth, ScreenHeight, Title);

  width := -1;
  height := -1;
  GetScreenSize(width, height);

  CloseGraph;

  WriteLn('width=', width, ' height=', height);
  if (width = ScreenWidth) and (height = ScreenHeight) then
    WriteLn('ok')
  else
    WriteLn('fail expected ', ScreenWidth, 'x', ScreenHeight);
end.
