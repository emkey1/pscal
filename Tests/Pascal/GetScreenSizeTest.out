width=320 height=240
ok
